src/components/Card/Card.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from "react";
+
+const Card = () => {
+
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+
+    function login(e) {
+        console.log(email,password)
+        e.preventDefault()
+        
+    }
+
+  return (
+    <>
+    
+    <form>
+      <section>
+        <div className="w-[464px] relative left-96 min-h-screen overflow-hidden mt-20">
+          <div className="w-full h-[700px] p-6 m-auto bg-[#F3F3F3] rounded-md shadow-md lg:max-w-xl">
+            <h1 className="text-3xl font-extrabold text-center text-blue-600 mb-4">
+              SouJunior
+            </h1>
+            <h2 className="text-center">Entrar</h2>
+            <hr className="my-8 h-px bg-gray-200 border-0 gray:bg-gray-700 mb-20" />
+            <form>
+              <div>
+                <label className="block text-sm font-semibold text-gray-800"></label>
+                <div className="relative flex w-full flex-wrap items-strech mb-3">
+                  <span className="absolute inset-y-[30%] right-[2%] z-10 h-full leading-snuf font-normal">
+                    {" "}
+                    <svg
+                      width="23"
+                      height="23"
+                      viewBox="0 0 23 23"
+                      fill="none"
+                      xmlns="http://www.w3.org/2000/svg"
+                    >
+                      <path
+                        d="M0.523375 6.7108L10.9262 0.4738C11.1011 0.368893 11.3012 0.313477 11.5051 0.313477C11.709 0.313477 11.909 0.368893 12.0839 0.4738L22.4766 6.71192C22.56 6.76189 22.629 6.83261 22.6769 6.9172C22.7248 7.00178 22.75 7.09734 22.75 7.19455V21.5001C22.75 21.7984 22.6315 22.0846 22.4205 22.2955C22.2095 22.5065 21.9234 22.6251 21.625 22.6251H1.375C1.07663 22.6251 0.790483 22.5065 0.579505 22.2955C0.368526 22.0846 0.25 21.7984 0.25 21.5001V7.19342C0.24998 7.09621 0.275153 7.00066 0.323064 6.91607C0.370975 6.83149 0.439989 6.76077 0.523375 6.7108ZM18.6393 8.27455L11.5686 14.3934L4.35287 8.2678L2.89713 9.9823L11.5821 17.3567L20.1107 9.97668L18.6393 8.27455V8.27455Z"
+                        fill="#E6E6E6"
+                      />
+                    </svg>
+                  </span>
+                  <input
+                    type="email"
+                    onChange={(e) => setEmail(e.target.value)}
+                    className="relative w-full px-4 py-2 mt-2 text-blue-700 border rounded-md focus:border-purple-400 focus:ring-blue-300 focus:outline-none focus:ring focus:ring-opacity-40"
+                    placeholder="E-mail"
+                  />
+                </div>
+              </div>
+
+              <div>
+                <label className="block text-sm font-semibold text-gray-800"></label>
+                <div className="relative">
+                  <span className="absolute inset-y-[20%] right-[2%] z-10 h-full leading-snuf font-normal">
```

Comment:
o correto é `leading-snug`. ver https://tailwindcss.com/docs/line-height
```suggestion
                  <span className="absolute inset-y-[20%] right-[2%] z-10 h-full leading-snug font-normal">
```